tools/clang/test/HLSLFileCheck/hlsl/linker/resources/preserve_cb_types.hlsl

```diff
@@ -0,0 +1,164 @@
+// RUN: %dxilver 1.7 | %dxc -T lib_6_5 -Fo dxl_6_5 %s | FileCheck %s -check-prefixes=CHECK,LIBGV_ORIG
+
+// Why is GV not mutated to handle for lib_6_6+?
```

Comment:
I feel like we should have an answer to that. Is it really the case?

Reply:
Created issue https://github.com/microsoft/DirectXShaderCompiler/issues/5202 to track it.